app/views/users/show.html.erb

```diff
@@ -1 +1,30 @@
-<h1>Here is the specified user</h1>
+<div>
+  <div class="user-container">
+    <%= image_tag(@user.photo + "images/rss.jpg", size: "50x50") %>
+
+    <div class="user-info">
+      <h5><%= @user.name %></h5>
+      <p>Number of posts : </p>
+    </div>
+
+  </div>
+  <div>
+    <div>
+      <h5>Bio</h5>
+      <p><%= @user.bio %></p>
+    </div>
+    <% @user.three_most_recent_posts.each do |post| %>
+      <div>
+        <%= link_to user_post_path(@user.id, post.id), class: "link-info" do %>
+          <h5><%= post.title %></h5>
+          <p><%= post.text %></p>
+        <% end %>
+      </div>
```

Comment:
- I would advise you to add `comment` and `like` in the post section. This is because your page should much a wireframe. example of codes: 
`<small>Comments: <%= post.comments_counter || 0 %>,</small>` 
`<small>Likes: <%= post.likes_counter || 0 %></small>`

- [ ] Wireframe
![Capture1](https://user-images.githubusercontent.com/63972670/218238542-fa77bca0-e140-474b-8338-1f746b41516d.JPG)